### PR TITLE
Workaround to use fuzzed_keys instead of minimized_keys for missing m…

### PR DESCRIPTION
**Pull request by alhijazi**
…inimzed testcases

### CODE REVIEW

File: src/clusterfuzz/_internal/bot/tasks/setup.py [hunk 1]
```diff
@@ -345,8 +345,16 @@ def _get_testcase_key_and_archive_status(testcase):
...
-    archived = bool(testcase.archive_state & data_types.ArchiveStatus.MINIMIZED)
-    return key, archived
+    # TODO(alhijazi): Remove this check once the issue with blob cleanup
+    # in minimize postprocess is resolved.
+    if storage.get(blobs.get_gcs_path(key)):
+      archived = bool(
+          testcase.archive_state & data_types.ArchiveStatus.MINIMIZED)
+      return key, archived
+    else:
+      logs.log(
```
Comment (alhijazi) [inline src/clusterfuzz/_internal/bot/tasks/setup.py:355]: @jonathanmetzman, I think we could even schedule a minimize_task here. WDYT?

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: src/clusterfuzz/_internal/bot/tasks/setup.py (hunk 1 above)
Comment (jonathanmetzman) [reply src/clusterfuzz/_internal/bot/tasks/setup.py]: WHy?

Comment (alhijazi) [reply src/clusterfuzz/_internal/bot/tasks/setup.py]: Minimize task won't be rescheduled for this issue. It had terminated properly and we had mistakenly deleted the minimized testcase. We could reschedule minimize here to regenerate the minimized testcase and that would be used later by progression task.

**Pull request merged**